prfromtransifex.py

```diff
@@ -208,12 +208,17 @@ def createNewPullRequest(g,
         debug("Changed files: %s", " ".join(os.linesep.split(changedfiles)))
     
         info("Things changed & force pushing")
-        debug(git["commit", "-m", pr_commit % {'mode': mode,
-                            'minpercent': minpercent,
-                            'langcount': len(files)}]())
-        
-        debug(git["push", "-f", "origin", wr_branch]())
+        commit_msg = pr_commit % {'mode': mode, 'minpercent': minpercent, 'langcount': len(files)}
+        debug(git["commit", "-m", commit_msg]())
+
     
+        if pr:
+            info("PR exists already, squashing changes to commit in PR")
+            debug(git["reset", "--soft", "HEAD~1"])
+            debug(git["commit", "-m", commit_msg])
```

Comment:
This will create a commit, then discard the commit with reset, and then commit again.

This seems wrong to me.

```suggestion

    
        if pr:
            info("PR exists already, squashing changes to commit in PR")
            debug(git["reset", "--soft", "HEAD~1"])
            debug(git["commit", "-m", commit_msg])
        else:
            debug(git["commit", "-m", commit_msg]())
```

Given that now they are the same command at the end, it could happen outside of the if else. Is there a significance to the difference of the existing vs non-existant `()`? Or is it simply missing in the new call on line 218?

Reply:
Yes moving this outside the if/else is fine.

The missing `()` is probably me having forgotten to put it where it belongs. I did not have a chance to test this, so it could very well be that this is indeed a syntax error.

Reply:
```suggestion

        if pr:
            info("PR exists already, squashing changes to commit in PR")
            debug(git["reset", "--soft", "HEAD~1"])

        debug(git["commit", "-m", commit_msg]())
```